Source/LibationWinForms/Form1.BackupCounts.cs

```diff
@@ -17,7 +17,9 @@ protected void Configure_BackupCounts()
 			beginPdfBackupsToolStripMenuItem.Format(0);
 
             LibraryCommands.LibrarySizeChanged += setBackupCounts;
-            LibraryCommands.BookUserDefinedItemCommitted += setBackupCounts;
+			//Pass null to the runner to get the whole library.
```

Comment:
Thank you for including comments like this. `(null, null)` is cryptic without it